apps/linode_helpers/roles/database/tasks/postgres.yml

```
@@ -0,0 +1,13 @@
+---
+- name: install postgres
+  apt:
+    pkg:
+      - postgresql
+
+- name: update postgres password
+  postgresql_query:
+    db: postgres
+    query: "ALTER USER postgres PASSWORD '{{ postgres_password }}';"
+  become: true
+  become_method: su
+  become_user: postgres
```

```suggestion
- name: update postgres password
  postgresql_query:
    db: postgres
    query: "ALTER USER postgres PASSWORD '{{ postgres_password }}';"
  become: true
  become_method: su
  become_user: postgres
  no_log: true
```
password shows in log